examples/CMakeLists.txt

```diff
@@ -4,9 +4,9 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0
 
-# No longer add cargo-without-cmake here.
-# This causes duplicate builds, as this crate is already entirely built
-# When using `cargo test`
```

Comment:
can you leave this comment here :-)

Reply:
huh i didnt even notice this got deleted